packages/fx-core/src/component/constants.ts

```diff
@@ -722,6 +722,23 @@ export const TabNewUIOptionItem: OptionItem = {
   ],
 };
 
+export const DashboardOptionItem: OptionItem = {
+  id: "dashboard-tab",
+  label: `$(browser) ${getLocalizedString("core.DashboardOption.label")}`,
+  description: getLocalizedString("core.Option.preview"),
+  cliName: "dashboard-tab",
+  detail: getLocalizedString("core.DashboardOption.detail"),
+  groupName: getLocalizedString("core.options.separator.scenario"),
+  data: "https://aka.ms/teamsfx-dashboard-app",
```

Comment:
Seems the link is not ready yet?

Reply:
Yeah, this link currently points to bing, we will change it after wiki page is ready.